cdk/src/open-data-platform/lambda/write-demographic-data-handler.ts

```diff
@@ -0,0 +1,233 @@
+// asset-input/src/open-data-platform/lambda/write-demographic-data-handler.js
```

Comment:
When do you think this lambda will be invoked? If it's not during `cdk deploy` then it's not really part of the deployment and shouldn't be in the `cdk` directory.

Reply:
Plan is to move to parallel directory called 'api' but in separate PR